GameDefinition/game-src/Client/UI/Content/Input.hs

```diff
@@ -62,18 +62,18 @@ standardKeysAndMouse = InputContentRaw $ map evalKeyDef $
   , ("space", ( [CmdMinimal, CmdMeta]
               , "clear messages and show history"
               , ExecuteIfClear LastHistory ))
-  , ("Tab", ( [CmdMove]
+  , ("Tab", ( [CmdMinimal]
```

Comment:
I've misled you: CmdMove should stay, too, so that it appears in the 'Terrain exploration' section as well as in the first section.